### PR TITLE
Add modal option for popovers

### PR DESCRIPTION
**Pull request by akrikos**
Adds a background color for modal popovers when `isModal` property is truthy

### CODE REVIEW

File: src/PopoverBackground.jsx [hunk 1]
```diff
@@ -18,6 +18,10 @@ class PopoverBackground extends React.Component {
       'zIndex': 999
     };
 
+    if (this.props.isModal) {
```
Comment (underbyte) [inline src/PopoverBackground.jsx]: use `classnames`

File: src/Popover.jsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ class Popover extends React.Component {
     let popover;
 
     this._backgroundDiv.style.display = 'block';
-    React.render(<PopoverBackground onRequestClose={this.props.onRequestClose} />, this._backgroundDiv);
+    React.render(<PopoverBackground isModal={ this.props.isModal } onRequestClose={this.props.onRequestClose} />, this._backgroundDiv);
```
Comment (adampickeral) [inline src/Popover.jsx]: add a propType for `isModal` at the bottom of this file

Comment (akrikos) [inline src/Popover.jsx]: ok

File: src/PopoverBackground.jsx (hunk 1 above)
Comment (akrikos) [inline src/PopoverBackground.jsx]: I'm not sure what you mean good sir. Please elaborate?

Comment (adampickeral) [inline src/PopoverBackground.jsx]: yeah, what do you mean? no classes are added here

Comment (underbyte) [inline src/PopoverBackground.jsx]: sorry I was looking at this wrong and thinking you were adding a class.

**Comment by adampickeral** [comment]
:+1:

**Comment by jdrush89** [comment]
:moneybag: :monkey:

**Pull request merged**